src/main/groovy/com/palantir/launchconfig/IdeaLaunchConfigTask.groovy

```diff
@@ -76,6 +76,16 @@ class IdeaLaunchConfigTask extends DefaultTask {
         ])
     }
 
+    /**
+     * Build the VM Params from the JVM Args and the System Properties.
+     */
+    private List<String> buildVmParams(JavaExec javaExec) {
```

Comment:
we're building the values for the `VM_PARAMETERS value, figured it should match.